requirements.txt

```diff
@@ -0,0 +1,2 @@
+jira
+jenkins
```

Comment:
what's this file doing at the top-level? 
if required by `jenkins_jira_integration.py` maybe both files can go into their own folder, e.g. `cassandra-builds/jenkins-jira-integration/`